src/params/travels.jl

```diff
@@ -0,0 +1,19 @@
+Base.@kwdef struct Holiday <: InfectionTravels
+    days::Float64 = 0
+    peak::Float64 = 0
+    height::Float64 = 0
+    minimum::Float64 = 0
+  end
+  
+  function (f::Holiday)(state::SimState, params::SimParams, infection_time::Real=0.0)
+    #@assert kind(event) == OutsideTransmissionEvent
+    days = (f.days/2)^2
+    rand(state.rng) < max((Float64(-abs(infection_time-f.peak)^2)*f.height)/days + f.height, f.minimum)
```

Comment:
please explain what is this function and perhaps name the function appropriately? if this is a parabola, maybe `ParabolicImports` would be a good name?

Reply:
yes, its good name